src/cljs/pyregence/pages/near_term_forecast.cljs

```diff
@@ -1,41 +1,42 @@
-   [pyregence.components.map-controls.camera-tool       :refer [camera-tool]]
-   [pyregence.components.map-controls.collapsible-panel :refer [collapsible-panel]]
-   [pyregence.components.map-controls.information-tool  :refer [information-tool]]
-   [pyregence.components.map-controls.legend-box        :refer [legend-box]]
-   [pyregence.components.map-controls.match-drop-tool   :refer [match-drop-tool]]
-   [pyregence.components.map-controls.measure-tool      :refer [measure-tool]]
-   [pyregence.components.map-controls.mouse-lng-lat     :refer [mouse-lng-lat]]
-   [pyregence.components.map-controls.scale-bar         :refer [scale-bar]]
-   [pyregence.components.map-controls.time-slider       :refer [time-slider]]
-   [pyregence.components.map-controls.tool-bar          :refer [tool-bar]]
-   [pyregence.components.map-controls.zoom-bar          :refer [zoom-bar]]
-   [pyregence.components.mapbox                         :as mb]
-   [pyregence.components.messaging                      :refer [message-box-modal
-                                                                toast-message!]]
-   [pyregence.components.nav-bar                        :refer [nav-bar]]
-   [pyregence.components.popups                         :refer [fire-popup]]
-   [pyregence.components.svg-icons                      :as svg]
-   [pyregence.config                                    :as c]
-   [pyregence.state                                     :as !]
-   [pyregence.styles                                    :as $]
-   [pyregence.utils.async-utils                         :as u-async]
-   [pyregence.utils.browser-utils                       :as u-browser]
-   [pyregence.utils.data-utils                          :as u-data]
-   [pyregence.utils.misc-utils                          :as u-misc]
-   [pyregence.utils.number-utils                        :as u-num]
-   [pyregence.utils.time-utils                          :as u-time]
-   [react                                               :as react]
-   [reagent.core                                        :as r]))
+   [cljs.core.async.interop                                                   :refer-macros [<p!]]
+   [clojure.core.async                                                        :refer [<! go]]
+   [clojure.edn                                                               :as edn]
+   [clojure.spec.alpha                                                        :as s]
+   [clojure.string                                                            :as str]
+   [cognitect.transit                                                         :as t]
+   [herb.core                                                                 :refer [<class]]
+   [pyregence.analytics                                                       :refer [gtag]]
+   [pyregence.components.map-controls.camera-tool                             :refer [camera-tool]]
+   [pyregence.components.map-controls.weather-station-observation-latest-tool :as weather-obs]
```

Comment:
If you rename `weather-station-observation-latest-tool` to `weather-station-tool` you'll need to update this line (and the indentation of the rest of the lines in this `require` section).